javascript-loops-practice/countdown.js

```diff
@@ -0,0 +1,8 @@
+/* exported countdown */
+function countdown(number) {
+  const array = [];
+  for (number; number >= 0; number--) {
```

Comment:
make sure to use `let` to declare the `number` in the first statement of the `for` loop